docs/website/assets/sass/docs.sass

```diff
@@ -1,5 +1,14 @@
 $panel-header-logo-width: 85%
 
+// this will add space before the target element so that the floating navbar
+// doesn't cover it when following an anchor link
+\:target::before
+  +desktop
+    content: ""
+    display: block
+    height: 3.75rem // height of the navbar + 0.5rem
```

Comment:
I think I'd increase this a bit...

<img width="574" alt="image" src="https://user-images.githubusercontent.com/870638/226319802-13ebe9c0-e4e0-4700-8e1e-ac1396be1462.png">

Feels like some extra space wouldn't hurt. But it's not a blocker, this is so much better than before. Also just a matter of preference, I guess.

Reply:
I've increased it to 4.25 now. I also realised that the margin wasn't needed.

One thing to note is that this space is inserted into the page so I was trying to keep it fairly small.

https://user-images.githubusercontent.com/1774239/226322111-8681c45b-37e0-4c14-9ec1-4c298bebfc58.mov

Reply:
(I'm not sure we have any other option as the dials on the markdown parser are limited)